data/common.yaml

```diff
@@ -117,6 +117,28 @@ scriptworker_users:
         - notary8
         - notary9
         - notary10
+  vpn:
+    cltbld:
+      home: /builds/scriptworker
+      supported_behaviors:
+        - mac_notarize_vpn
+      notarization_users:  # Do we need this?
+        - notary1
+      dmg_prefix: "prod"
+      cot_product: "mozillavpn"
+      keychain_filename: "mozilla.20220330.keychain"
+      widevine_filename: "N/A"
+      ed_key_filename: "N/A"
```

Comment:
If we want to e.g. beetmove a signed binary, we'll need to CoT-sign our CoT artifact, so we'll need this specified for prod mac signers.